### ISSUE TITLE
Extraneous xlink namespace in Execute Output Reference

### ISSUE THREAD

**Issue by billaj** [3liz/py-qgis-wps#3]
In Execute responses,  Output References have a "xlink:href" attribute :
```
<wps:ExecuteResponse xmlns:gml="http://www.opengis.net/gml" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_response.xsd" service="WPS" version="1.0.0" (...)>
(...)
  <wps:ProcessOutputs>
    <wps:Output>
      <ows:Identifier>Output</ows:Identifier>
      <ows:Title>Ruggedness map</ows:Title>
      <ows:Abstract>Ruggedness map</ows:Abstract>
      <wps:Reference xlink:href="http://(server)/ows/?MAP=(map).qgs&amp;service=WMS&amp;request=GetCapabilities&amp;layer=Output" mimeType="application/x-ogc-wms"/>
    </wps:Output>
  </wps:ProcessOutputs>
</wps:ExecuteResponse>
```
According to OGC specs, Output References should have a href without a namespace :
```
<complexType name="OutputReferenceType">
<annotation>
<documentation>Reference to an output value that is a web accessible resource.</documentation>
</annotation>
<attribute name="href" type="anyURI" use="required">
<annotation>
<documentation>Reference to a web-accessible resource that is provided by the process as output. This attribute shall contain a URL from which this output can be electronically retrieved. </documentation>
</annotation>
</attribute>
<attributeGroup ref="wps:ComplexDataEncoding"/>
</complexType>
```

(the Input Reference processing is fine, contrary to the Outputs they are indeed meant to have a xlink:href)

**Comment by dmarteau** [comment]
Fixed in d9f0d0bf1d688b7c284c9d984d7bbbd2a1f51a56

Note that according to old discussion (http://lists.opengeospatial.org/pipermail/wps-dev/2011-February/000096.html) this may be a bug/inconsistency of the spec.

**Comment by billaj** [comment]
Successfully tested, thanks.
Interesting link...the use of an unqualified href only in the Execute response sure is confusing.
